src/app/tabs/newAdmin/subdomains/operations.js

```diff
@@ -165,9 +177,9 @@ export const setSubdomainOwner = (
   const label = `0x${sha3(normalize(subdomain))}`;
   const node = namehash(parentDomain);
 
-  const rns = getRNS();
-  await rns.compose();
-  rns.contracts.registry.methods.setSubnodeOwner(node, label, newAddress)
+  const r = getRNS();
+  await r.compose();
+  r.contracts.registry.methods.setSubnodeOwner(node, label, newAddress)
```

Comment:
and this?

Reply:
These are not part of the flow, will make the change in another issue.